Assets/MixedRealityToolkit/_Core/Inspectors/Profiles/MixedRealityPointerProfileInspector.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
+
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.InputSystem;
+using UnityEditor;
+using UnityEditorInternal;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Inspectors
+{
+    [CustomEditor(typeof(MixedRealityPointerProfile))]
+    public class MixedRealityPointerProfileInspector : MixedRealityBaseConfigurationProfileInspector
```

Comment:
summary tag

Reply:
We've agreed not to do documentation on inspectors for now, as they are fairly self explanatory.
Also, they are not part of the API, they are internal to unity.